streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -76,13 +62,14 @@ public V apply() {
         this.streamsGraphNode = streamsGraphNode;
     }
 
-    <T> KTable<K, T> build(final KStreamAggProcessorSupplier<K, ?, V, T> aggregateSupplier,
-                           final String functionName,
-                           final StoreBuilder storeBuilder,
-                           final boolean isQueryable) {
+    <KR, T> KTable<KR, T> build(final KStreamAggProcessorSupplier<K, KR, V, T> aggregateSupplier,
+                                final String functionName,
+                                final StoreBuilder<? extends StateStore> storeBuilder,
```

Comment:
There's no need to discard the parameterized type, since we can express "builder for any kind of state store" using a type bound.